student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
+    super
+    @classroom = classroom
+  end
```

Comment:
- [ ] Please add the inheritance parameters to the `initialize` method and `super` see this [example](http://ruby-for-beginners.rubymonstas.org/advanced/private_methods.html). 
[Hint]: It's supposed to be like that:
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```